rust/tests/integration/core/example.rs

```diff
@@ -22,19 +22,19 @@ use futures::{StreamExt, TryStreamExt};
 // EXAMPLE END MARKER
 use serial_test::serial;
 // EXAMPLE START MARKER
-use typedb_driver::{
-    answer::{
-        concept_document::{Leaf, Node},
-        ConceptRow, QueryAnswer,
-    },
-    concept::{Concept, ValueType},
-    Error, TransactionType, TypeDBDriver,
-};
+use typedb_driver::{answer::{
```

Comment:
The format of this import looks strange. Pretty sure that our `rustfmt` will change it

Reply:
(you can try running the `rustfmt` command line from `.factory`)